### ISSUE TITLE
3.1.0 not working with yarn 1.13.0 on windows

### ISSUE THREAD

**Issue by ezpuzz** [peerigon/updtr#79]
updtr --version : 3.1.0
yarn version 1.13.0-20181004.1525

```
❯ .\node_modules\.bin\updtr
Running updtr with custom configuration:

o use: yarn

Looking for outdated modules...
> yarn outdated --json --flat
  .

 ERROR  Error when trying to parse stdout from command 'yarn outdated --json --flat': Could not find object with type === table
    at yarnParser (E:\repos\virtuoso\node_modules\updtr\dist\index.js:1665:11)
    at parse$1.yarn.outdated.stdout (E:\repos\virtuoso\node_modules\updtr\dist\index.js:1710:22)
    at Object.outdated (E:\repos\virtuoso\node_modules\updtr\dist\index.js:1609:14)
    at getUpdateTasksFromStdout (E:\repos\virtuoso\node_modules\updtr\dist\index.js:879:3)
    at E:\repos\virtuoso\node_modules\updtr\dist\index.js:915:28
    at Generator.throw (<anonymous>)
    at asyncGeneratorStep (E:\repos\virtuoso\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _throw (E:\repos\virtuoso\node_modules\@babel\runtime\helpers\asyncToGenerator.js:29:9)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

output of `yarn outdated --json --flat`:
```
{"type":"info","data":"Color legend : \n \"<red>\"    : Major Update backward-incompatible updates \n \"<yellow>\" : Minor Update backward-compatible features \n \"<green>\"  : Patch Update backward-compatible bug fixes"}
{"type":"table","data":{"head":["Package","Current","Wanted","Latest","Package Type","URL"],"body":[["@ember-decorators/babel-transforms","2.1.1","2.1.2","2.1.2","devDependencies","https://github.com/ember-decorators/babel-transforms#readme"],["@neighborly/ui","0.0.22","0.0.22","0.1.0","devDependencies","https://github.com/neighborly/neighborly-ui#readme"],["@types/ember","3.0.20","3.0.24","3.0.24","devDependencies","https://github.com/DefinitelyTyped/DefinitelyTyped.git"],["@types/ember__test-helpers","0.7.3","0.7.5","0.7.5","devDependencies","https://github.com/DefinitelyTyped/DefinitelyTyped.git"],["@types/ember-data","3.0.2","3.1.1","3.1.1","devDependencies","https://github.com/DefinitelyTyped/DefinitelyTyped.git"],["@types/ember-qunit","3.4.1","3.4.2","3.4.2","devDependencies","https://github.com/DefinitelyTyped/DefinitelyTyped.git"],["@types/ember-test-helpers","1.0.2","1.0.3","1.0.3","devDependencies","https://github.com/DefinitelyTyped/DefinitelyTyped.git"],["ember-cli-babel","6.17.2","6.17.2","7.1.2","devDependencies","https://github.com/babel/ember-cli-babel#readme"],["ember-cli-htmlbars","3.0.0","3.0.1","3.0.1","devDependencies","https://github.com/ember-cli/ember-cli-htmlbars"],["ember-cli-htmlbars-inline-precompile","1.0.3","1.0.4","1.0.4","devDependencies","https://github.com/ember-cli/ember-cli-htmlbars-inline-precompile#readme"],["ember-cli-typescript","1.4.3","1.4.4","1.4.4","devDependencies","https://github.com/typed-ember/ember-cli-typescript"],["ember-decorators","2.5.0","2.5.2","2.5.2","devDependencies","https://ember-decorators.github.io/ember-decorators"],["ember-power-select","2.0.8","2.0.9","2.0.9","devDependencies","http://www.ember-power-select.com"],["ember-source","3.4.3","3.4.5","3.4.5","devDependencies","https://emberjs.com/"],["eslint-plugin-prettier","2.7.0","2.7.0","3.0.0","devDependencies","https://github.com/prettier/eslint-plugin-prettier#readme"],["husky","1.0.1","1.1.1","1.1.1","devDependencies","https://github.com/typicode/husky#readme"],["sass","1.14.0","1.14.1","1.14.1","devDependencies","https://github.com/sass/dart-sass"],["typescript","3.0.3","3.1.1","3.1.1","devDependencies","http://typescriptlang.org/"],["typescript-eslint-parser","19.0.1","19.0.2","19.0.2","devDependencies","https://github.com/eslint/typescript-eslint-parser"]]}}
```

I'm on Windows 10 and running in powershell.

**Comment by ezpuzz** [comment]
https://github.com/yarnpkg/yarn/blob/eb2b565bb9b948e87b11119482ebc184a9d66141/src/reporters/json-reporter.js#L22

yarn hardcodes a `\n` here:    ```stdout.write(`${JSON.stringify({type, data})}\n`);```

**Comment by ezpuzz** [comment]
node.js `console.log` also uses `\n` so I suggest checking for `\n` instead of `os.EOL`

**Comment by tannerbaum** [comment]
I wish I had seen this several months back when attempting to upgrade the dependencies and noticing that our tests were failing on Windows! It was tricky to track down what you had already discovered @ezpuzz. The issue should be resolved now with #86 , thanks for posting the issue.
